parsec/core/fs/remote_loader.py

```diff
@@ -417,20 +425,51 @@ async def load_manifest(
             FSDeviceNotFoundError
             FSInvalidTrustchainError
         """
+        return await self._load_manifest(
+            entry_id,
+            version=version,
+            timestamp=timestamp,
+            expected_backend_timestamp=expected_backend_timestamp,
+        )
+
+    async def _load_manifest(
+        self,
+        entry_id: EntryID,
+        version: Optional[int] = None,
+        timestamp: Optional[DateTime] = None,
+        expected_backend_timestamp: Optional[DateTime] = None,
+        workspace_entry: Optional[WorkspaceEntry] = None,
+    ) -> BaseRemoteManifest:
         if timestamp is not None and version is not None:
             raise FSError(
```

Comment:
I would have raised a `ValueError` or a `RuntimeError` (or use an assert !) instead given this should never occur and it result from a bug in the code (hence we want it not be catched !)

Reply:
5769bda